python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/ip-nat-part1.j2

```diff
@@ -12,7 +12,7 @@ ip nat translation address selection any
 {%     if ip_nat.translation.address_selection.hash_field_source_ip | arista.avd.default(false) %}
 ip nat translation address selection hash field source-ip
 {%     endif %}
-{%     for timeout in ip_nat.translation.timeouts | arista.avd.default([]) %}
+{%     for timeout in ip_nat.translation.timeouts | arista.avd.default([]) | arista.avd.natural_sort %}
```

Comment:
You already did  the changes for ip-nat-part1.j2 in another PR

Reply:
Removed this.

Reply:
update the PR description

Reply:
Done